backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/po/POTrlInfo.java

```diff
@@ -47,8 +47,15 @@
 	 */
 	private final boolean translated;
 
+	private final boolean translations;
```

Comment:
what's the purpose of this flag?

Reply:
this flag is true if the `POTrlInfo` is related to translation table

Reply:
the purpose of POTrlInfo is to describe a base table, tell which is the trl table, what columns are translated etc,

and not to have a POTrlInfo for each table, no matter if is a base table or a trl table